### PR TITLE
add a 'rake clean' and make it a dependency for every compile

### PR DESCRIPTION
**Pull request by yithian**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -27,3 +30,9 @@ task :compile do
     end
   end
 end
+
+task :clean do
```
Comment (komidore64) [inline Rakefile]: we can really simplify `rake clean` here because it's a mechanism built into `rake`.

```ruby
CLEAN.include(@dest)
```

**Comment by yithian** [comment]
Well if you want to use things I probably should have known but didn't, then sure!

Reading up a little bit now, this should probably be CLOBBER not CLEAN, right?

**Comment by komidore64** [comment]
either

**Comment by komidore64** [comment]
yeah, that looks a lot nicer.

ACK

**Pull request merged**